pkg/kstatus/polling/statusreaders/common.go

```diff
@@ -161,7 +161,7 @@ func getNamespaceForNamespacedResource(object runtime.Object) string {
 
 // keyForNamespacedResource returns the object key for the given identifier. It makes
 // sure to set the namespace to default if it is not provided.
-func keyForNamespacedResource(identifier wait.ResourceIdentifier) types.NamespacedName {
+func keyForNamespacedResource(identifier object.ObjMetadata) types.NamespacedName {
```

Comment:
Should this function go in the new object package?

Reply:
Will look at this in a separate PR.